gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild

```
@@ -0,0 +1,59 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=8
+inherit meson xdg
+
+DESCRIPTION="A window switcher, run dialog and dmenu replacement"
+HOMEPAGE="https://github.com/lbonn/rofi"
+SRC_URI="https://github.com/lbonn/rofi/releases/download/${PV}%2Bwayland1/rofi-${PV}+wayland1.tar.xz"
+S=${WORKDIR}/rofi-${PV}+wayland1
```

please put variables like ${PV} and ${WORKDIR} in " like this:
"${WORKDIR}"

```suggestion
S="${WORKDIR}"/rofi-"${PV}"+wayland1
```